4-projects/README.md

```diff
@@ -39,21 +94,109 @@ Change the `BRANCH_NAME` from `development` to `non-production` or `production`
+   cp -RT ../terraform-example-foundation/4-projects/ .
+   ```
+1. Copy the Jenkinsfile script to the root of your new repository.
+   ```
+   cp ../terraform-example-foundation/build/Jenkinsfile .
+   ```
+1. Update the variables located in the `environment {}` section of the `Jenkinsfile` with values from your environment:
+    ```
+    _TF_SA_EMAIL
+    _STATE_BUCKET_NAME
+    _PROJECT_ID (the cicd project id)
+    ```
+1. Copy terraform wrapper script  to the root of your new repository.
+   ```
+   cp ../terraform-example-foundation/build/tf-wrapper.sh .
+   ```
+1. Ensure wrapper script can be executed.
+   ```
+   chmod 755 ./tf-wrapper.sh
+   ```
+1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap.
+1. Rename `shared.auto.example.tfvars` to `shared.auto.tfvars` and update the file with values from your environment and bootstrap.
+1. Rename `development.auto.example.tfvars` to `development.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_d_shared_restricted`.
+1. Rename `non-production.auto.example.tfvars` to `non-production.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_n_shared_restricted`.
+1. Rename `production.auto.example.tfvars` to `production.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_p_shared_restricted`.
+1. Commit changes.
+   ```
+   git add .
+   git commit -m 'Your message'
+   ```
+1. Push your plan branch.
+   ```
+   git push --set-upstream origin plan
+   ```
+    - Assuming you configured an automatic trigger in your Jenkins Master (see [Jenkins sub-module README](../0-bootstrap/modules/jenkins-agent)), this will trigger a plan. You can also trigger a Jenkins job manually. Given the many options to do this in Jenkins, it is out of the scope of this document see [Jenkins website](http://www.jenkins.io) for more details.
+1. Review the plan output in your Master's web UI.
+1. Merge changes to production branch.
+   ```
+   git checkout -b production
+   git push origin production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. After production has been applied, apply development and non-production.
+1. Merge changes to development branch.
+   ```
+   git checkout -b development
+   git push origin development
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Merge changes to non-production branch.
+   ```
+   git checkout -b non-production
+   git push origin non-production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. You can now move to the instructions in the step [4-projects](../4-projects/README.md).
```

Comment:
```suggestion
1. You can now move to the instructions in the step [5-app-infra](../5-app-infra/README.md).
```

Reply:
@bharathkkb should this line be moved to the Cloud Build section, since step 5-app-infra uses Cloud Build ?

Reply:
Yeah I think we should move to CB and also swap CB to be the first instruction.